c++/src/capnp/CMakeLists.txt

```diff
@@ -168,6 +168,7 @@ if(NOT CAPNP_LITE)
   set_target_properties(capnp_tool PROPERTIES CAPNP_INCLUDE_DIRECTORY
     $<JOIN:$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/..>,$<INSTALL_INTERFACE:${CMAKE_INSTALL_BINDIR}/..>>
   )
+set_target_properties(capnp_tool PROPERTIES COMPILE_FLAGS -DCAPNP_INCLUDE_DIR='"${CMAKE_INSTALL_FULL_INCLUDEDIR}"')
```

Comment:
nitpick: Seems like this should be indented to match the surrounding lines?

Reply:
ahh ugly, sorry, am I blind or did vim betray me few minutes before push?